tests/Application/src/Foundry/Story/MoreBooksStory.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Foundry\Story;
+
+use App\Foundry\Factory\BookFactory;
+use function Zenstruck\Foundry\Persistence\flush_after;
+use Zenstruck\Foundry\Story;
+
+final class MoreBooksStory extends Story
+{
+    public function build(): void
+    {
+        flush_after(function () {
```

Comment:
maybe should I use that
[[DX][Internal] use Foundry for comic books](https://github.com/Sylius/SyliusResourceBundle/commit/68b78b402c4b4621c34de43dbfede553d8b53efe)